netops_demo/py/functions/generate/generate.py

```diff
@@ -29,12 +33,8 @@ def generate(context, event):
 
 def init_context(context):
 
-    # initialize context structure
-    for context_attr in ['state', 'configuration', 'manager']:
-        setattr(context.user_data, context_attr, None)
-
-    # start in idle state
-    context.user_data.state = 'idle'
+    # Init states
+    libs.utils.utils.init_context(context)
```

Comment:
`init_context` being a shared function would mean that _all_ functions ever added here have the same init_context which is not true. I would not worry about this init_context being duplicated across functions